src/ai/ai.cpp

```diff
@@ -413,6 +423,44 @@ void ShipAI::runOrders()
             owner->orderRoaming();  //Nothing to dock, just fall back to roaming.
         }
         break;
+
+        case AI_Repair:            //Repair at the nearest station
+        if (owner->docking_state == DS_NotDocking)
+        {
+            P<SpaceStation> friendly_station = searchFriendlyStation(15000.0);
```

Comment:
Would it be a good idea to tie the range to the server's long-range radar setting and/or the ship's jump range, instead of a fixed 15U?